src/plugins/management/public/components/management_sections.tsx

```diff
@@ -17,61 +17,88 @@
  * under the License.
  */
 
-import React from 'react';
-import { EuiFlexGroup, EuiFlexItem, EuiToolTip, EuiIcon } from '@elastic/eui';
-
+import { i18n } from '@kbn/i18n';
 import { ManagementSectionId } from '../types';
 
-interface ManagementSectionTitleProps {
-  text: string;
-  tip: string;
-}
+const injestTitle = i18n.translate('management.sections.ingestTitle', {
```

Comment:
Typo here and on line 27: "injest" -> "ingest". Might be worth a quick grep to search for this typo elsewhere.